docs/core/tools/dotnet-nuget-push.md

```diff
@@ -69,7 +69,7 @@ The command pushes an existing package. It doesn't create a package. To create a
 
 - **`--skip-duplicate`**
 
-  When pushing multiple packages to an HTTP(S) server, treats any 409 Conflict response as a warning so that the push can continue.
+  When pushing multiple packages to an HTTP(S) server, treats any 409 Conflict response as a warning so that the other push(s) can continue.
```

Comment:
```suggestion
  When pushing multiple packages to an HTTP(S) server, treats any 409 Conflict response as a warning so that other pushes can continue.
```